Modules/Filtering/ImageGrid/wrapping/itkWarpImageFilter.wrap

```diff
@@ -1,5 +1,16 @@
 itk_wrap_class("itk::WarpImageFilter" POINTER)
-  foreach(s ${WRAP_ITK_SCALAR})
-    itk_wrap_image_filter_combinations("${s}" "${s}" "${WRAP_ITK_VECTOR_REAL}" 2+)
+  foreach(curr_dim ${ITK_WRAP_IMAGE_DIMS})
+    if(${curr_dim} LESS 1)
+        continue()
+    endif()
+    # Vector components dimension must match image dimension. Temporarily change the global vector dims list.
+    set(BACKUP_ITK_WRAP_VECTOR_COMPONENTS "${ITK_WRAP_VECTOR_COMPONENTS}")
```

Comment:
How about a more unique name (e.g., that includes `WARP_IMAGE_FILTER_`) rather than `BACKUP_`?  If someday someone else happens to use `BACKUP_` in a different file that is effectively inside or containing this loop, we'd have a heck of a time debugging that one overwrote the other.

Reply:
See a few lines lower where this variable is unset  so that it has very limited scope.

Reply:
Agreed, chances are that `itk_wrap_image_filter_combinations` (between the set and unset) does not and will not use a variable named `BACKUP_ITK_WRAP_VECTOR_COMPONENTS`.  Likewise agreed, chances are that cmake code before this is executed is not and will not be setting that variable for use after this code is executed.